l10n_be_mis_reports/script/generate_xml.py

```diff
@@ -0,0 +1,385 @@
+                    compare_method_field.setAttribute('name', 'compare_method')
+                    value = xml_document.createTextNode('pct')
+                    compare_method_field.appendChild(value)
+                    record.appendChild(compare_method_field)
+
+                    # Create field 'style_id' and add as record's child
+                    # TODO : not generic for the profit and loss
+                    style_id_field = xml_document.createElement('field')
+                    style_id_field.setAttribute('name', 'style_id')
+                    if rubcode in SPECIALSTYLES:
+                        style_id_field.setAttribute('ref', f"mis_report_style_l10n_be_2i")
+                    elif rubcode == '9905':
+                        style_id_field.setAttribute('ref', f"mis_report_style_l10n_be_1")
+                    else:
+                        if(report_id == PLREPORTID):
+                            style_id_field.setAttribute('ref', f"mis_report_style_l10n_be_{set_indent(calc_file, rubcode, 2, rubcode_list)}")
+                        else:
+                            style_id_field.setAttribute('ref', f"mis_report_style_l10n_be_{set_indent(calc_file, rubcode, 1, rubcode_list)}")
+                    record.appendChild(style_id_field)
+
+                    # Create field 'auto_expand_accounts' and 'auto_expand_accounts_style_id' and add as record's child
+                    if(needs_auto_expand(calc)):
+                        auto_expand_field = xml_document.createElement('field')
+                        auto_expand_field.setAttribute('name', 'auto_expand_accounts')
+                        auto_expand_field.setAttribute('eval', 'True')
+                        record.appendChild(auto_expand_field)
+
+                        auto_expand_style_field = xml_document.createElement('field')
+                        auto_expand_style_field.setAttribute('name', 'auto_expand_accounts_style_id')
+                        auto_expand_style_field.setAttribute('ref', 'mis_report_style_l10n_be_acc_det')
+                        record.appendChild(auto_expand_style_field)
+
+                    # Create field 'sequence' and add as record's child
+                    sequence_field = xml_document.createElement('field')
+                    sequence_field.setAttribute('name', 'sequence')
+                    value = xml_document.createTextNode(f"{sequence}")
+                    sequence += 10
+                    sequence_field.appendChild(value)
+                    record.appendChild(sequence_field)
+
+                    odoo.appendChild(record)
+    return xml_document
+
+def get_label_value(account_table):
+    """Get the name of an account """
+    label = ''
+    for i in range(5):
+        try:
+            label = account_table[i]['labels']['en']  
+        except KeyError:
+            pass
+        if len(label) > 0:
+            break
+    return label
```

Comment:
```suggestion
def _get_account_label(account_table, lang="en"):
    """Get the name of an account """
    label = ''
    for i in range(5):
        try:
            label = account_table[i]['labels'][lang]  
        except KeyError:
            pass
        if len(label) > 0:
            break
    return label
```

* remove the try except (you can use the get() function available in Python dictionaries)
* what are you trying to check with `len(label) > 0`?
* what if nothing is found? Shouldn't you at least output a warning?
* Where does `range(5)` come from?